tests/kafkatest/version.py

```diff
@@ -120,7 +120,7 @@ def get_version(node=None):
         return DEV_BRANCH
 
 DEV_BRANCH = KafkaVersion("dev")
-DEV_VERSION = KafkaVersion("3.6.0-SNAPSHOT")
+DEV_VERSION = KafkaVersion("3.7.0-SNAPSHOT")
```

Comment:
forgot to dump the dev-version -- this breaks the system test setup.

Reply:
This may be just bad timing as @satishd is releasing 3.6.0 right now. We typically bump this version after the vote passes according to the release process: https://cwiki.apache.org/confluence/display/KAFKA/Release+Process#ReleaseProcess-Afterthevotepasses

I wonder if we should undo this change from this PR and let Satish do it as part of the release process. WDYT?

Reply:
The vote was closed, and @satishd did pump the version in all other places already (cf https://github.com/apache/kafka/commit/9e3b1f9b9bf48603acde7f71c704af812a6dab4b) -- this PR should have also bumped this version (otherwise system test start to break).

`DEV_VERSION` must match what is in `gradle.properties`, so this is really just a plain fix :)